code/datums/martial/cqc.dm

```diff
@@ -50,15 +50,14 @@
 		return NONE
 	if(attack_type == PROJECTILE_ATTACK)
 		return NONE
-	if(!prob(block_chance))
-		return NONE
```

Comment:
Should probably check that the user has enough stamina to block without being stamcrit, so people don't just floor themselves